src/cdk/v1/heap/mapping/track.yaml

```diff
@@ -18,8 +18,9 @@
     type: flatJson
 
 - destKey: timestamp
-  sourceKeys: timestamp
-  sourceFromGenericMap: true
+  sourceKeys:
+    - timestamp
+    - originalTimestamp
```

Comment:
Can we please mention these heap changes too in description if they are valid changes?

Reply:
@yashasvibajpai These changes are coming as the develop was not up to date with main. After updating this change is not part of this commit.